### DISCUSSION TITLE
Account Menu for VSCode Extensions

### DISCUSSION THREAD

**Discussion by sameemqiitm** [microsoft/vscode-discussions#1704] [Extension Development QnA] [answered]
Hey , im working on a vs code extension, I am interested in using the account menu that appears in the bottom left side. Where we have the Account with the Authentication Provider listed . (Please refer the Image Attached)
![accounts menu](https://github.com/user-attachments/assets/e52102a6-5415-47f0-bbdb-615788f8f7d0)
I actually want to implement what happens on clicking sign out from here .
How to do that???

**Comment by isidorn** [comment, marked as answer]
Use the AuthenticationSession API
https://github.com/microsoft/vscode/blob/main/src/vscode-dts/vscode.d.ts#L17104
